eth-denver/welcome.md

```diff
@@ -0,0 +1,117 @@
+
+Welcome to the first official Cosmos Burner Chain! Following in the tradition of the Burner Wallet project (and other temporary low-security chains) we've built and launched a game chain to be used for the duration of the Eth Denver Hackathon. The goal of this chain is to provide a blueprint for your own application specific blockchain that interacts with other EVM based networks like Ethereum and xDai.
+
+The Eth Denver Cosmos Burner Chain consists of a few moving pieces. The first of which is the Cosmos SDK itself. This is a framework, much like Ruby-on-Rails, for building new blockchains. It comes with a set of standard modules that handles a common set of features: Bank (Fungible Tokens like ERC-20), Auth (Accounts secured by public key cryptography), Proof-of-Stake + Slashing (together these modules handle a proof of stake validator set utilizing Tendermint Consensus with slashing conditions for equivocation and downtime) as well as Params (live updates to the chain) and Gov (a governance module that can be used for text proposals, param changes and directing a community pool of funds).
+
+Out of the core SDK modules, our Burner Chain uses Bank, Auth, Proof-of-Stake and Slashing. Outside of the standard fare it also uses Peggy, a new Module that was developed by Swish Labs with support from the Intechain Foundation. This module is essentially a bridge between EVM chains like Ethereum and xDai. It is used to transport Fungible Tokens and has recently been extended to support the transport of Non-Fungible Tokens as well (an integration with the Ethermint EVM module would allow arbitrary message transportation if you're looking for a cool hackathon project!). Our burner chain also uses the NFT Module which is in the "incubator" stage and hosted within the cosmos/modules repository. The final piece of our chain is the Scavenge Module, which is part of the sdk-tutorials repo. This module allows you to post a riddle with a bounty for whoever can solve it (You can also check out the Scavenge tutorial to learn how to build a custom module for the Cosmos SDK.)
+
+In summary the pieces that are included are:
+ * Bank
+ * Auth
+ * Proof-of-Stake
+ * Slashing
+ * NFT
+ * Peggy (oracle, ethbridge, nftbridge)
+ * Scavenge
+
+The purpose of the chain is to extend the Scavenger hunt game in order to interact with the Eth Denver BuffiDai game (If you haven't heard about BuffiDai take a look here). Cosmos has 1200 XP to give out and 2 NFT badges (Big Brain and Cosmos Brain). We have locked all of them up within a peggy bridge contrat deployed to the xDai chain. This contract is controlled by the validators of the Burner Chain which relay transactions between the two chains. In order to claim XP on the Cosmos Burner Chain you must sign up for our newsletter. Afterwards you will receive a balance of XP on the Cosmos burner chain where you will be able to move the XP back to the BuffiDai contract on xDai using the Peggy ETH bridge.
```

Comment:
> … peggy bridge contrat deployed …

Typo: `contract`

> In order to claim XP on the Cosmos Burner Chain you must sign up for our newsletter. Afterwards you will receive…

Change to: `…you must complete our survey. Upon completion, you'll be entitled to receive…`